### ISSUE TITLE
Mask input to EfficientSam models

### ISSUE THREAD

**Issue by carlosuc3m** [mit-han-lab/efficientvit#62]
Hello,
First of all congrats on a great job.
I am playing around with your models, trying to get the mask input prompts to work and I have not been able to get good results.

I am using resized masks. First I resize the long side to 256 and then pad the bottom right of the image, I am following the approach of the following issue on the original SAM https://github.com/facebookresearch/segment-anything/issues/169

What am I doing wrong?

Thanks a lot for your help!
CArlos

**Comment by GIS-PanTianLi** [comment]
Have you solved this problem? I think their model do not support masks as prompt.

**Comment by carlosuc3m** [comment]
Nope, probably it was not trained with mask prompts. SAme for negative point prompts.
The best approach is to sample random points from the mask evaluate several times and take a mean or the one with best certainty.
